README-WIN.md

```diff
@@ -86,9 +86,9 @@ SSH key management in WSL can be done with `keychain`.  Here is an example of
 what to add to `.bashrc`:
 
 ```shell
-/usr/bin/keychain -q --nogui $HOME/.ssh/private_key1
+/usr/bin/keychain -q --nogui $HOME/.vagrant.d/insecure_private_key
 /usr/bin/keychain -q --nogui $HOME/.ssh/private_key2
-source $HOME/.keychain/$HOST-sh
+source $HOME/.keychain/$HOSTNAME-sh
```

Comment:
Can you double check this?  I'm pretty sure $HOST is usually defined but not $HOSTNAME.

Reply:
@mw2q it is set to `$HOSTNAME` or can use `hostname` on Ubuntu 20.04/22.04

Reply:
Maybe it's better to use $(hostname) since $HOST and $HOSTNAME can vary is the distro feels like changing their default environment.